### DISCUSSION TITLE
Project tests and CI/CD pipelines

### DISCUSSION THREAD

**Discussion by cjlapao** [Parallels/rq-dashboard#442] [General]
We now have the automation pipelines ready but the tests are failing, looking for volunteers to make this also work. we will then be able to be more confident that a pr will not break the project

**Comment by Srabani5** [comment]
Hi cjlapao, I am willing to do this. However, I was trying to run the application using Docker. It seems like the docker image needs to be updated.

**Reply by cjlapao** [reply]
@Srabani5 I will have a look at this and create an issue for us to track. I am also creating a project so we can track all of this

**Reply by cjlapao** [reply]
I have checked, and we have a docker image with the latest release working fine. We also have a docker-compose file to start a new one locally. could you let me know a bit more what is the issue you are facing?
